### PR TITLE
Update compound icons

### PR DESCRIPTION
**Pull request by alfogrillo**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -9,7 +9,7 @@ let package = Package(
         .library(name: "Compound", targets: ["Compound"])
     ],
     dependencies: [
-        .package(url: "https://github.com/vector-im/compound-design-tokens.git", revision: "387d2b7211f07761c67e849c59414a1bb803defa"),
+        .package(url: "https://github.com/vector-im/compound-design-tokens.git", from: "0.0.7"),
```
Comment (pixlwave) [inline Package.swift:12]: Nice, thank you for switching to a tag 🙏

File: Sources/Compound/Icons/CompoundIcons.swift [hunk 2]
```diff
@@ -35,8 +35,8 @@ public struct CompoundIcons {
     public let visibilityOff = compound.iconVisibilityOff
     public let videoCall = compound.iconVideoCall
     public let videoCallOff = compound.iconVideoCallOff
-    public let videoCallMissed = compound.iconVideoCallMissed
-    public let videoCallDeclined = compound.iconVideoCallDeclined
+    public let videoCallMissed = compound.iconVideoCallMissedSolid
+    public let videoCallDeclined = compound.iconVideoCallDeclinedSolid
```
Comment (pixlwave) [inline Sources/Compound/Icons/CompoundIcons.swift:39]: Would you mind updating the properties to include the `Solid` suffix and handle the changes in EX please?

Also, I think there are a couple of other breaking changes like this with what was solid will now be an outline. Let me find them and DM you.

Comment (alfogrillo) [reply Sources/Compound/Icons/CompoundIcons.swift]: > Would you mind updating the properties to include the Solid suffix and handle the changes in EX please?

Applied [here](https://github.com/vector-im/compound-ios/pull/41/commits/fac005e9ec7d3c1f51176f4d79238cc2a7dedc5e)

**Review by pixlwave — APPROVED** [review]
LGTM 👍

**Pull request merged**